File: test/req.js
#!/usr/bin/env node
var util = require ( "util" ) ;
var gepard = require ( "gepard" ) ;
var name   = gepard.getProperty ( "name", "ack" ) ;
var c      = gepard.getClient() ;
c.request ( name,
  function(e) // The event is sent -> end connection and exit
  {
  	console.log ( util.inspect ( e, { showHidden: false, depth: null } ) ) ;
    this.end() ;
  }
);

File: python/test/req.py
#!/usr/bin/env python

import os, sys
sys.path.insert ( 0, os.path.dirname(os.path.abspath(__file__) ) + "/../" )

import gepard

name   = gepard.util.getProperty ( "name", "ack" ) ;
client = gepard.Client.getInstance()

def freq ( e ):
	print ( e.getStatus() )
	e.getClient().close()

client.request ( name, freq )
